templates/cluster-template-aws.rc

```diff
@@ -6,6 +6,11 @@ export WORKER_MACHINE_COUNT=1
 # AWS region
 export AWS_REGION="eu-west-1"
 
+# AWS image
+# Use clusterawsadm ami list to verify there is a AMI that
+# matches your desired base os and kubernetes version.
+export AWS_IMAGE_LOOKUP_BASE_OS=
```

Comment:
I like setting the `AWS_IMAGE_LOOKUP_BASE_OS` here. Since we are setting the kubernetes version, I do think it would be okay so set a default ubuntu image here too. I'd prefer that over not having it work by default. 
We could leave a note at the kubernetes version, if you change this value you may have to change the base os image too.

Reply:
This is not needed in the end.